firebase_admin/db.py

```diff
@@ -125,40 +125,91 @@ def child(self, path):
         full_path = self._pathurl + '/' + path
         return Reference(client=self._client, path=full_path)
 
-    def get(self):
-        """Returns the value at the current location of the database.
+    def _get_with_etag(self):
+        """Returns the value at the current location of the database, along with its ETag.
+        """
+        data, headers = self._client.request('get', self._add_suffix(),
+                                             headers={'X-Firebase-ETag' : 'true'},
+                                             resp_headers=True)
+        etag = headers.get('ETag')
+        return etag, data
+
+    def get_etag(self):
```

Comment:
Perhaps call it `etag()`? That sounds more consistent with the other methods we have in this module like `child()` and `reference()`.